tests/run-pass/current_dir_with_isolation.stderr

```diff
@@ -0,0 +1,12 @@
+warning: `getcwd` was made to return an error due to isolation
+  |
+  = note: run with -Zmiri-isolation-error=warn to get warning with a backtrace
+  = note: run with -Zmiri-isolation-error=ignore to hide warning
+  = note: run with -Zmiri-disable-isolation to disable isolation
```

Comment:
The point of the "nobacktrace" option is to not clutter the screen so much... so is it really a good idea to print all these notes?
They are not printed with `=warn`, are they?

Reply:
Agreed, it's a bit verbose. But given that it's printed only once out of N `getcwd` ops, I thought it might be user-friendly to print notes about the possible actions. I can drop the notes if you like.
Yes, they are not printed with `=warn`. But note that `=warn` prints a trace per line where `getcwd` is called.

Reply:
`warn` also goes through the rustc diagnostic system, so shouldn't it deduplicate?

I find it unlikely that someone would set this to `warn-nobacktrace` without having read the docs, and if they have read the docs then they will know the other options.

Reply:
I'm not sure how exactly deduping works, but my guess is -- since `warn` option prints a span that includes a location, only the calls from a given file+line location get deduped. Vs. `warn-nobacktrace` doesn't print a span, so all the calls from different locations are considered duplicates (because of the same message content?) and hence deduped.

I started with the intention of having default behaviour (`warn-nobacktrace` earlier) less verbose but with enough info about possible actions. Now that we've moved to default behaviour being `abort`, yes, it makes less sense to print notes. I'll drop the notes.

Reply:
> I started with the intention of having default behaviour (warn-nobacktrace earlier) less verbose but with enough info about possible actions. Now that we've moved to default behaviour being abort, yes, it makes less sense to print notes. I'll drop the notes.

Ah, yes, that makes sense. If we ever make this the default, we should reconsider this. :)